src/docs/environment/index.mdx

```diff
@@ -293,3 +299,24 @@ In your sentry virtualenv:
 pip uninstall uwsgi
 pip install --no-cache-dir uwsgi
 ```
+
+---
+
+**Problem:** `DoesNotExist: Subscription matching query does not exist`
+
+**Solution:** In getsentry run the following to mock a subscription:
+
+```bash
+./bin/mock-subscription <org_slug>
+```
+
+---
+
+**Problem:** Something like `Error: No such container: sentry_postgres`
+
+**Solution:** Review the [bootstrap services](/environment/#bootstrap-services)
```

Comment:
Dang, I remember markdown link notation didn't work before, but it works on the preview.